src/api/payments/controller.ts

```
@@ -2,9 +2,28 @@ import zalo from "@/services/zalopay";
 import { Request, Response } from "express";
 
 const createZaloOrder = async (req: Request, res: Response) => {
-  const { userid, items, amount } = req.body;
+  const { userid, items } = req.body;
+
+  if (
+    !Array.isArray(items) ||
+    !items.every(
+      (item) =>
+        item.hasOwnProperty("itemname") &&
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)